### PR TITLE
Apply coding styles (aka "phase 2")

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Apply coding styles rules on PrestaShop + travis build
| Type?         | improvement
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Travis build should be green.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9468)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

**Comment by mickaelandrieu** [comment]
I need help here, I don't understand why Travis don't start: do I have something invalid with my update?

**Comment by PierreRambaud** [comment]
Never close a PR to restart a build. You must use the Travis interface for that :/
Don't know if travis have an interface to trigger build from PR or if you need to use their cli tool.

**Comment by mickaelandrieu** [comment]
> Never close a PR to restart a build. You must use the Travis interface for that :/

there is no mention of this pull request in the Travis interface.

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
@eternoendless Let's go? :D

File: classes/Access.php
Comment (eternoendless) [inline classes/Access.php:3]: Why is it adding dots?

File: classes/Combination.php
Comment (eternoendless) [inline classes/Combination.php:98]: It's misaligning aligned statements

File: classes/Access.php
Comment (PierreRambaud) [reply classes/Access.php:3]: because of this rule `phpdoc_annotation_without_dot [@Symfony]` I think

File: classes/Combination.php
Comment (mickaelandrieu) [reply classes/Combination.php:98]: or maybe he's aligning? probably a space/tab conflict :+1:

Comment (mickaelandrieu) [reply classes/Combination.php:98]: in fact, looking at the class (and not at the diff), it looks cleaner

clean

**Comment by mickaelandrieu** [comment]
and we can't merge it anymore lol,

to whom who accepts to merge this: please rebase and re-apply the command, thanks!

**Comment by PierreRambaud** [comment]
@mickaelandrieu Your fault https://github.com/PrestaShop/PrestaShop/pull/9450 :trollface:

**Comment by mickaelandrieu** [comment]
not really, it's because @eternoendless is reviewing all the files :troll:

would you mind to rebase and merge it asap? the more we wait, the painful it will be.

Or we block every merge: alternative option :)

File: classes/Access.php
Comment (eternoendless) [reply classes/Access.php:3]: It's messing with our file headers :\

File: classes/Combination.php
Comment (eternoendless) [reply classes/Combination.php:98]: It's a matter of opinion, but I liked it better before:

screen shot 2018-08-22 at 14 51 51

**Comment by eternoendless** [comment]
I'm not reviewing everything, I suspected I'd find some issues by reading the first few files, and I did 🙂

File: classes/Combination.php
Comment (mickaelandrieu) [reply classes/Combination.php:98]: maybe there is an option for it in php cs fixer but we didn't add it

Comment (PierreRambaud) [reply classes/Combination.php:98]: Personally prefer the new style, no need IDE configuration and prevent useless add / remove line in git history when creating a new line.
That's maybe why Symfony / PSR2 did this.

**Comment by mickaelandrieu** [comment]
it's rebased :+1:

**Comment by mickaelandrieu** [comment]
> It's a matter of opinion, but I liked it better before:

I'm not opposed to improvements, but *after* we have merged this one ;)

Now we will have to comply with the standards, I'm pretty sure we will have to discuss together soon, to improve some rules and maybe remove other ones.

**Comment by eternoendless** [comment]
Ok for merge after tests are green.

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @mickaelandrieu and everyone for this =)

**Pull request merged**
